docs/userguide/dependency_management.rst

```diff
@@ -309,4 +309,4 @@ In some cases, you might need to specify the minimum required python version.
 This is handled with the ``python_requires`` keyword supplied to ``setup.cfg``
 or ``setup.py``.
 
-Example WIP
\ No newline at end of file
+Example WIP
```

Comment:
Did you change the EOF here? I think it's better to not touch it.

Reply:
It's GitHub online editor's behavior

Reply:
Well, if it wasn't there, it's probably okay to have it because it's a good idea to have all test files end with LF. Honestly, it's a kind of thing that would be great to have in a linter.